entity-registry/src/main/java/com/linkedin/metadata/aspect/hooks/OwnershipOwnerTypes.java

```diff
@@ -0,0 +1,120 @@
+package com.linkedin.metadata.aspect.hooks;
+
+import static com.linkedin.metadata.Constants.OWNERSHIP_ASPECT_NAME;
+
+import com.linkedin.common.*;
+import com.linkedin.common.urn.Urn;
+import com.linkedin.metadata.aspect.ReadItem;
+import com.linkedin.metadata.aspect.RetrieverContext;
+import com.linkedin.metadata.aspect.batch.ChangeMCP;
+import com.linkedin.metadata.aspect.plugins.config.AspectPluginConfig;
+import com.linkedin.metadata.aspect.plugins.hooks.MutationHook;
+import com.linkedin.util.Pair;
+import java.util.*;
+import java.util.stream.Stream;
+import javax.annotation.Nonnull;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Getter
+@Setter
+@Accessors(chain = true)
+public class OwnershipOwnerTypes extends MutationHook {
+  @Nonnull private AspectPluginConfig config;
+
+  protected Stream<Pair<ChangeMCP, Boolean>> writeMutation(
+      @Nonnull Collection<ChangeMCP> changeMCPS, @Nonnull RetrieverContext retrieverContext) {
+    List<Pair<ChangeMCP, Boolean>> results = new LinkedList<>();
```

Comment:
Not a blocker to merge, this can be expressed more simply as
```
return changeMCPS.stream()
    .map(item -> aspectFilter(item)
        ? Pair.of(item, processOwnershipAspect(item))
        : Pair.of(item, false));
```

Reply:
Thanks. will do